automan/cluster_manager.py

```diff
@@ -368,6 +377,14 @@ def update(self, rebuild=True):
                 if rebuild:
                     self._rebuild(host, home)
 
+    def delete(self, sim_dir):
+        for worker in self.workers:
+            host = worker.get('host')
+            home = worker.get('home')
+            nfs = worker.get('nfs', False)
+            if host != 'localhost' and not nfs:
```

Comment:
What about locally?

Reply:
user can delete the folders in local machine manually, but for remote he has to go to each location to delete the output folders incase he need to do so.